src/core/shape/2d_primitives.js

```diff
@@ -504,17 +921,22 @@ p5.prototype.point = function(...args) {
 };
 
 /**
- * Draws a quad to the canvas. A quad is a quadrilateral, a four-sided
- * polygon. Some examples of quads include rectangles, squares, rhombuses,
- * and trapezoids. The first pair of parameters (`x1`,`y1`) sets the quad's
- * first point. The following pairs of parameters set the coordinates for
- * its next three points. Parameters should proceed clockwise or
- * counter-clockwise around the shape.
+ * Draws a quadrilateral (four-sided shape).
```

Comment:
In the one-line description, can we change it to "four-sided polygon" to keep consistency with the long description?

Reply:
@Qianqianye I actually removed the word polygon from all the shape descriptions. Happy to restore it, but this seemed simpler, if less precise.